src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
+    if (!isNaN(value)) {
```

Comment:
isNaN can have unexpected behavior and not so understanded
```suggestion
    if (value !== undefined) {
```

Reply:
Thanks!
Code updated✅